region/crateria/west/Gauntlet Entrance.json

```diff
@@ -348,24 +387,58 @@
       "link": [2, 1],
       "name": "Base",
       "requires": [
-        "ScrewAttack"
+        "ScrewAttack",
+        {"or": [
+          "canCarefulJump",
+          {"ammo": {"type": "Super", "count": 1}},
+          {"acidFrames": 100}
```

Comment:
This room seems fine, but I see that the 1->2 equivalent of this strat has far fewer frames, is that right?

Reply:
This block is for a yapping maw since they are harder to avoid from this side.
[1,2] is for avoiding acid at the wall right of center blockade since it disrupts your movement.
I think I'll add the same small acidFrame count here.  I do like expecting a bit of damage crossing the room.